newIDE/app/src/GameDashboard/GameDetailsDialog.js

```diff
@@ -278,7 +286,7 @@ export const GameDetailsDialog = ({
                       alignItems="center"
                       noMargin
                     >
-                      {authorUsernames && (
+                      {authorUsernames && ownerUsernames && (
```

Comment:
it may be an edge case, but I don't think we need the ownerUsernames here, only the authorUsernames.
the check on ownerUsernames can be done below, only when you want to add the crown icon